### PR TITLE
[i18n-KO] Translated quicktour page to Korean

### PR DESCRIPTION
**Pull request by wonhyeongseo**
# What does this PR do?

Translated the `Quicktour.mdx` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?
@sgugger, @ArthurZucker, @eunseojo may you please review this PR?

### CODE REVIEW

**Review by sgugger — APPROVED** [review]
LGTM for the general structure. @ArthurZucker if you could give this a quick proof, that would be awesome!

Thanks a lot for your PR!

File: docs/source/ko/quicktour.mdx [hunk 1]
```diff
@@ -0,0 +1,537 @@
...
+>>> from torch import nn
+
+>>> pt_predictions = nn.functional.softmax(pt_outputs.logits, dim=-1)
+>>> print(pt_predictions)
+tensor([[0.0021, 0.0018, 0.0115, 0.2121, 0.7725],
+        [0.2084, 0.1826, 0.1969, 0.1755, 0.2365]], grad_fn=<SoftmaxBackward0>)
+```
+</pt>
+<tf>
+🤗 Transformers provides a simple and unified way to load pretrained instances. This means you can load an [`TFAutoModel`] like you would load an [`AutoTokenizer`]. The only difference is selecting the correct [`TFAutoModel`] for the task. For text (or sequence) classification, you should load [`TFAutoModelForSequenceClassification`]:
```
Comment (ArthurZucker) [inline docs/source/ko/quicktour.mdx:267]:
```suggestion
```
This part is duplicated 😉

**Review by ArthurZucker — APPROVED** [review]
Also looks good to me! Minus a duplicated part

**Comment by wonhyeongseo** [comment]
Thank you so much for your review @sgugger and especially @ArthurZucker for your awesome proof-read! I have commited your suggestion. Although it is a bit late, happy lunar new year, everyone! 🐇 🌕

**Comment by ArthurZucker** [comment]
Happy new year! 🐰

**Pull request merged**